OpenRA.Mods.Common/Traits/Attack/AttackBase.cs

```diff
@@ -55,6 +55,10 @@ public abstract class AttackBaseInfo : PausableConditionalTraitInfo
 		[Desc("Tolerance for attack angle. Range [0, 512], 512 covers 360 degrees.")]
 		public readonly WAngle FacingTolerance = new(512);
 
+		[Desc("Show the target cursor even on empty terrain cells when enabled.",
+			"Any traits implementing IMove must be absent or disabled for this to work.")]
+		public readonly bool TargetsTerrain = false;
```

Comment:
```suggestion
		public readonly bool TargetsTerrain = false;
```

We were discussing on discord that this needs a rename. It's almost sounds as if by having it false the unit won't be able to target terrain at all

Personally I don't have suggestions, sorry

Reply:
Changed to `TargetTerrainWithoutForceFire`. Any better?